### ISSUE TITLE
[트랙2] Matric 확인 부탁드립니다.

### ISSUE THREAD

**Issue by dearyonghoon** [AI-GrandChallenge/round-1#46]
수정 완료 하셨다는 #37 Matric이 LeaderBoard에 적용이 되지 않는 것 같습니다. 모든 스코어가  0.3129 입니다.

**Comment by hwany-j** [comment]
네 확인하였습니다. 구현 중 데이터셋의 포맷이 맞지 않는 버그가 있어서 수정 후 다시 오늘 중으로 공지하도록 하겠습니다. 불편을 끼쳐드려 죄송합니다.

**Comment by hoondental** [comment]
baseline 코드가 실행하면 에러가 나는데요... 그래서 모델이 어떤 형식으로 출력을 해야 하는지 유추를 할 수가 없습니다. 
실행이 되는 베이스라인 코드를 올려주시거나 출력 형식을 명확하게 알려주시기 바랍니다.

**Comment by hwany-j** [comment]
> baseline 코드가 실행하면 에러가 나는데요... 그래서 모델이 어떤 형식으로 출력을 해야 하는지 유추를 할 수가 없습니다.
> 실행이 되는 베이스라인 코드를 올려주시거나 출력 형식을 명확하게 알려주시기 바랍니다.

모델 형식은 자유롭게 짜셔도 됩니다. 모델 output을 Infer함수의 Return함수만 형식에 맞게 고쳐주시면 됩니다. 
infer 함수의 return dataframe의 형식은 아래와 같습니다. 

<img width="417" alt="스크린샷 2020-07-28 오후 1 28 38" src="https://user-images.githubusercontent.com/57430984/88619462-8f102480-d0d6-11ea-917c-fa2bf6092a79.png">

복수 구간의 경우, file_name을 두 번 쓰고, 각 구간별 start 와 end를 넣으시면 됩니다. 
예시 : 
file_name   | start | end 
a.wav          | 1       |   2 
a.wav          | 3      |  4
